Js/index.js

```diff
@@ -4,24 +4,170 @@ let btnCancelMobileMenu = document.querySelector("#btn-close-nav");
 const PortfolioClicked = document.querySelector(".link-portfolio");
 const AboutClicked = document.querySelector(".link-about");
 const ContactClicked = document.querySelector(".link-contact");
+const allBtnPopups = document.querySelectorAll(".btn-pop");
 
-var page = document.getElementsByTagName('body')[0];
-
+var page = document.getElementsByTagName("body")[0];
+var popup = document.getElementsByClassName("desk-work-list");
 
 function displaymobilemenu() {
-  //alert("changed");
   mobileMenu.classList.replace("mobile-menu-hide", "mobile-menu-show");
-  page.classList.add('noscroll');
+  page.classList.add("noscroll");
 }
 
 function hidemobilemenu() {
-  //alert("changed");
```

Comment:
- [ ] Good job fixing the (x) problem, great! However, there's still some unused code showing up in your code, as the previous reviewer said, it would be great to remove it as this is a best practice for clean coding.

Reply:
Actually haven't configured JS linter yet.
now will be the last review of mine . :)

Reply:
I haven't been able to see any unused code ..